File: src/main/java/ni/edu/uam/michimaker_api/dto/TransformacionDto.java
package ni.edu.uam.michimaker_api.dto;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class TransformacionDto {

    private Integer id;

    private String nombreFiltro;

    private String fecha;

    private String rutaImagen;

    private String imagenBase64;

    private Integer usuarioId;

    private String leyenda;
}

File: app/src/main/java/ni/edu/uam/michimaker/dto/TransformacionDto.kt
package ni.edu.uam.michimaker.dto

data class TransformacionDto(

    val id: Int? = null,

    val nombreFiltro: String,

    val fecha: String,

    val rutaImagen: String,

    val usuarioId: Int? = null,

    val leyenda: String? = null,

    val imagenBase64: String? = null
)